<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordToDeckForm</class>
 <widget class="DialogWindow" name="WordToDeckForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zkanji</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Select parts of words to add to the study deck:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="DeckFormListView" name="wordsTable">
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="ZStatusBar" name="listStatus"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="DefinitionWidget" name="defEditor" native="true"/>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZStatusBar</class>
   <extends>QStatusBar</extends>
   <header>zstatusbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DialogWindow</class>
   <extends>QMainWindow</extends>
   <header>dialogwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DefinitionWidget</class>
   <extends>QWidget</extends>
   <header>definitionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DeckFormListView</class>
   <extends>QTableView</extends>
   <header>wordtodeckform.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
